kasa/protocol.py

```diff
@@ -17,6 +17,10 @@
 from pprint import pformat as pf
 from typing import Dict, Generator, Optional, Union
 
+# When support for cpython older than 3.11 is dropped
```

Comment:
Adjusted comment since `asyncio.timeout` is in cpython 3.11+ but `asyncio.wait_for` is only fixed in cpython 3.12+